README.md

```diff
@@ -336,6 +336,26 @@ If custom CORS origin(s) are required it can be set here separated by `,` (comma
 
 **Mandatory** the OAuth client secret
 
+    session_management_ip_stack_api_token
+
+**Mandatory** The IP address lookup is required to select the closest cloudXR region. The token must be obtained on [this](https://ipstack.com/) website
+
+    session_management_adaptive_instance_scaling_enabled
+
+Enables adaptive instance scaling. This defaults to `True`
+
+    session_management_adaptive_instance_scaling_user_scale_ratio
```

Comment:
documentation is not matching the code. The default is `'true'` (stringified, lowercase version of True). It should be `true` (actual boolean value). Regarding the stringified version, not sure if ASP.NET Core can handle both `'True'` and `'true'`, better double check.